reviews three/swagger.yaml

```diff
@@ -0,0 +1,954 @@
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: Draft Orders API
+  x-logo:
+    url: "https://andrewjtech123.github.io/imagehost/shopify_logo_white.png"
+    backgroundColor: "#7ab55c"
+  description: >
+    # Draft Orders API 
+      
+      A [Draft Order](https://help.shopify.com/manual/orders/create-orders) represents the status of an order before the customer has completed the checkout process. Using the Draft Orders API you can create a draft order and transition it to a completed order with great flexibility.  The following flows are supported:
+      
+      * Create a draft order that calculates taxes and totals but accept payment from the customer outside of Shopify.
+      * Create a draft order (with taxes and totals) and email your customer a web checkout link to complete purchase and transition to a completed order.
+      * Use Draft Orders API in conjunction with the [Checkouts API](https://help.shopify.com/api/tutorials/sell-through-the-checkout-api) by created a Checkout object based on a draft order (Sales Channel SDK only).  This enables a highly customizable experience for channels.
+      
+     ## About Draft Orders
+    
+     When using the [Orders API](https://help.shopify.com/api/reference/order) to create orders, no payment information is calcualted and no transaction is performed.  With the Draft Orders API, on the other hand, you can create orders on which shipping, taxes and all total amounts are calculated by Shopify.  This enables a wide range of use cases including applying one-off discounts to orders, offering special wholesale pricing, and returning draft order invoices for order printing.
+        
+
+  
+schemes:
+  - https
+  
+host: your-store-domain.myshopify.com
+
+paths:
+  /admin/draft_orders.json:
+    get:
+      summary: Retrieve draft orders
+      description: You can use this endpoint to return a list of draft order objects.  This can be useful for a variety of applications including draft order printing.  For `line_items` you can use the value of the `custom` field (true/false) to differentiate between custom line items and product variant line items.
```

Comment:
why is the line_item description here instead of on the line_item field? 

`For `line_items` you can use the value of the `custom` field (true/false) to differentiate between custom line items and product variant line items.`